nixos/modules/services/web-servers/minio.nix

```diff
@@ -18,9 +18,9 @@ in
     };
 
     dataDir = mkOption {
-      default = "/var/lib/minio/data";
-      type = types.path;
-      description = "The data directory, for storing the objects.";
+      default = [ "/var/lib/minio/data" ];
+      type = types.listOf types.path;
```

Comment:
Isn't this a breaking change? This should at least be documented in the changelog.

Reply:
Yes, it is. Any other necessary things besides the changelog? I'm not really familiar with all the infrastructure, so will be grateful for any hints.

Reply:
For me it would be fine if it is documented in the changelog

Reply:
It took me a while to find it :)